src/main/java/duke/Task.java

```diff
@@ -0,0 +1,76 @@
+package duke;
+
+/**
+ * This class encapsulates the attributes, states and capabilities
+ * of a task: its name, its done status and the ability to change it.
+ *
+ * @author Siau Wee
+ *
+ */
+public class Task {
+
+    /** The name of the task. */
+    private String taskName;
```

Comment:
```suggestion
    private final String taskName;
```
Suggesting addition of `final` modifier.